dts/arm/nxp/nxp_rt6xx_common.dtsi

```diff
@@ -406,6 +406,7 @@
 	interrupts = <42 0>;
 	#address-cells = <1>;
 	#size-cells = <0>;
+	status = "disabled";
```

Comment:
why is this disabled, at least add comment in commit message.  Something like 'flexspi will be board specific modules of FLASH so leave it to the board to enable...'